copernicusmarine/command_line_interface/group_get.py

```diff
@@ -150,15 +150,15 @@ def cli_get() -> None:
     is_flag=True,
     default=False,
     help=documentation_utils.GET["SYNC_HELP"],
-    mutually_exclusive=["no-directories", "skip-existing"],
+    mutually_exclusive=["skip-existing"],
 )
 @click.option(
     "--sync-delete",
     cls=MutuallyExclusiveOption,
     is_flag=True,
     default=False,
     help=documentation_utils.GET["SYNC_DELETE_HELP"],
-    mutually_exclusive=["no-directories", "skip-existing"],
+    mutually_exclusive=["skip-existing", "no-directories"],  # noqa: E501
```

Comment:
```suggestion
    mutually_exclusive=["skip-existing", "no-directories",],
```
and then save in your IDE